.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 # pyenv
 .python-version
 .vscode/
+
+# resource group txt created for ext_sequencing scenario in e2e pipeline
+resource_groups_to_delete.txt
```

Comment:
The combinator creates a resource group and VMSS for the extension sequencing scenario and writes the resource group name to this file. Later, a task in the pipeline runs to cleanup the resources listed in this file.